src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -295,10 +295,13 @@ def instance(userid: Optional[str] = None,
 
                     # Iterate through _daemons.values() to find the daemon with the
                     # matching zonegroup_id
```

Comment:
This is one of the dangers of comments. 
The code this comment is refering to is changing but the comment is not.
We will just need one more refactor around this code to have this comment be completly missleading.

A part from that I think this comment is not adding value since the code it's refering to is as self explanatory as the comment itself.

Personaly, I would take the chance to remove it.

Reply:
Updated